x-pack/plugins/observability/public/components/shared/exploratory_view/hooks/use_lens_attributes.ts

```diff
@@ -37,54 +33,50 @@ export const getFiltersFromDefs = (
   });
 };
 
-export const useLensAttributes = ({
-  seriesId,
-}: Props): TypedLensByValueInput['attributes'] | null => {
-  const { getSeries } = useSeriesStorage();
-  const series = getSeries(seriesId);
-  const { breakdown, seriesType, operationType, reportType, dataType, reportDefinitions = {} } =
-    series ?? {};
+export const useLensAttributes = (): TypedLensByValueInput['attributes'] | null => {
+  const { allSeriesIds, allSeries } = useSeriesStorage();
 
-  const { indexPattern } = useAppIndexPatternContext();
+  const { indexPatterns } = useAppIndexPatternContext();
 
   return useMemo(() => {
-    if (!indexPattern || !reportType || isEmpty(reportDefinitions)) {
+    if (isEmpty(indexPatterns) || isEmpty(allSeriesIds)) {
       return null;
     }
 
-    const dataViewConfig = getDefaultConfigs({
-      reportType,
-      dataType,
-      indexPattern,
-    });
+    const layerConfigs: LayerConfig[] = [];
+
+    allSeriesIds.forEach((seriesIdT) => {
+      const seriesT = allSeries[seriesIdT];
```

Comment:
I've been wondering this whole time what 'T' stands for on a lot of variable naming so I guess I'll ask. Is this intended to mean `type`?

Reply:
it's my way of typing temporary :D